deployments/ansible/roles/collector/README.md

```diff
@@ -105,8 +126,50 @@ how to use the role in a playbook with minimal required configuration:
 - `splunk_fluentd_config`: Path to the fluentd config file on the remote host.
   (**default:** `/etc/otel/collector/fluentd/fluent.conf`)
 
+## Windows Specific Role Variables
+
+- `collector_path_win`: Default path of Splunk-otel-collector in windows 
+  (**default:** `C:\\Program Files\Splunk\OpenTelemetry Collector\`)
```

Comment:
The MSI installation directory is not currently configurable and installs to `%PROGRAMFILES%\Splunk\OpenTelemetry Collector`.